src/components/UI/CircleIcon.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div :class="$style.circle" :style="styles.icon">
+    <icon
+      :class="$style.icon"
+      :name="props.name"
+      :mdi="props.mdi"
+      :width="props.width"
+      :height="props.height"
+      :color="props.color"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive } from '@vue/composition-api'
+import { makeStyles } from '@/lib/styles'
+import Icon from '@/components/UI/Icon.vue'
+
+const useStyles = (props: {
+  background: string
+  width: number
+  height: number
+}) => {
+  return reactive({
+    icon: makeStyles(theme => ({
+      background: props.background,
+      width: `${props.width}px`,
+      height: `${props.height}px`
+    }))
+  })
+}
+
+type Props = {
+  color: string
+  background: string
+  name: string
+  mdi?: boolean
+  width: number
+  height: number
```

Comment:
```suggestion
  innerSize: number
  size: number
```
ここは`Icon.vue`とインターフェイスを近くしたいです